client/src/components/ApplicationDashBoard/ApplicationDashBoard.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import { useAppSelector } from "../../app/hooks";
+
+interface IStatusCount {
+  status: string;
+  count: number;
+}
+
+const ApplicationDashboard = (): JSX.Element => {
+  const [totalApplications, setTotalApplications] = useState(0);
+  const [applicationsByStatus, setApplicationsByStatus] = useState<
+    IStatusCount[]
+  >([]);
+  const user = useAppSelector((state) => state.user.userData);
+
+  useEffect(() => {
+    async function fetchAggregatedData() {
+      try {
+        const response = await axios.get(
+          `/api/applications/aggregated-user-stats?user_id=${user?._id}`
+        );
+        setTotalApplications(response.data.totalApplications);
+        setApplicationsByStatus(response.data.applicationsByStatus);
+      } catch (error) {
```

Comment:
It would be best to add null checks with API calls. While, we are the ones designing the API and creating the response, it is a good practice just in case. 

In this scenario, the application would crash if `response.data` is null

Reply:
Good catch @JimmyTran48  - New code pushed up!